### PR TITLE
Update listing entities and dto models with properties for company logo id and date for last apply

### PR DESCRIPTION
**Pull request by KristofferHogberg**

### CODE REVIEW

File: src/GT.Core/DTO/Impl/ListingOverviewDTO.cs [hunk 1]
```diff
@@ -20,5 +20,6 @@ public class ListingOverviewDTO : IGTDataTransferObject
 		public bool? FTE { get; set; }
 		public DateTime? CreatedDate { get; set; }
 		public string? ExperienceLevel { get; set; }
+		public string? ApplicationDeadline { get; set; }
```
Comment (RasmusBroborg) [inline src/GT.Core/DTO/Impl/ListingOverviewDTO.cs:23]: Should be of datatype 'Datetime?', same as CreatedDate.

File: src/GT.Data/Data/GTAppDb/Entities/Listing.cs [hunk 2]
```diff
@@ -33,5 +33,8 @@ public class Listing : IGTEntity
 		public ICollection<ListingInquiry>? Inquiries { get; set; }
 
 		public ExperienceLevel? ExperienceLevel { get; set; }
+
+		public DateTime? ApplicationDeadline { get; set; }
+		public string? CompanyLogoId { get; set; }
```
Comment (RasmusBroborg) [inline src/GT.Data/Data/GTAppDb/Entities/Listing.cs:38]: Should maintain line breaks between each property for conformity with the rest of the document.

File: src/GT.Data/GT.Data.csproj [hunk 3]
```diff
@@ -16,10 +16,5 @@
 		</PackageReference>
 		<PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="6.0.2" />
 	</ItemGroup>
-
-	<ItemGroup>
-	  <Folder Include="Migrations\GTAppMigrations\" />
-	  <Folder Include="Migrations\GTIdentityMigrations\" />
-	</ItemGroup>
```
Comment (RasmusBroborg) [inline src/GT.Data/GT.Data.csproj:23]: Why were these removed?

File: src/GT.Core/DTO/Impl/ListingDTO.cs [hunk 4]
```diff
@@ -47,5 +47,12 @@ public class ListingDTO : IGTDataTransferObject
 		/// </summary>
 		[StringLength(100)]
 		public string? ExperienceLevel { get; set; }
+
+		/// <summary>
+		/// Maps to an ApplicationDeadline name in the database.
+		/// </summary>
+		public DateTime? ApplicationDeadline { get; set; }
+
+		public string? CompanyLogoId { get; set; }
```
Comment (RasmusBroborg) [inline src/GT.Core/DTO/Impl/ListingDTO.cs:56]: Should perhaps have a StringLength limit of 450?

**Review by RasmusBroborg — CHANGES_REQUESTED** [review]

File: src/GT.Core/DTO/Impl/ListingDTO.cs (hunk 4 above)
Comment (OscarThorburn) [reply src/GT.Core/DTO/Impl/ListingDTO.cs:57]: Fixed limit

File: src/GT.Data/Data/GTAppDb/Entities/Listing.cs (hunk 2 above)
Comment (OscarThorburn) [reply src/GT.Data/Data/GTAppDb/Entities/Listing.cs:40]: Fixed formatting

File: src/GT.Data/GT.Data.csproj (hunk 3 above)
Comment (OscarThorburn) [reply src/GT.Data/GT.Data.csproj:23]: They should not have been. Restored

File: src/GT.Data/GT - Backup.Data.csproj [hunk 5]
```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```
Comment (RasmusBroborg) [inline src/GT.Data/GT - Backup.Data.csproj:3]: New file should not be included in PR.

File: src/GT.Data/GT.Data.csproj (hunk 3 above)
Comment (RasmusBroborg) [reply src/GT.Data/GT.Data.csproj:23]: Issue still remains after new commit.

**Review by RasmusBroborg — CHANGES_REQUESTED** [review]

**Review by RasmusBroborg — APPROVED** [review]
